modules/main_Presenter/client/index.jsx

```diff
@@ -13,43 +13,39 @@ import Code from 'sub_SharingCode/client/index'
 import AudienceList from 'sub_AudienceList/client/index'
 import Presentations from 'db/Presentations'
 
+import { CircularProgress } from 'material-ui'
+
+
 let Presenter = React.createClass({
-  mixins: [ReactMeteorData],
+  // mixins: [ReactMeteorData],
 
   componentWillMount() {
-    let user = Meteor.user()._id;
-
-    let link = this.props.data.link;
-    let gid = this.props.data.gid;
-
-    Meteor.call('createPresentation', link, user, gid, function (err, result) {
-      if (err) {
-        console.error('from preview ', err);
-      } else {
-        console.log('success!', this.props.params.gid);
-        // react.props.setPresentation(gid);
-        // window.open('/projector/' + gid);
-      }
-    })
   },
 
   componentDidMount() {
     const context = this
-    Presentation.find({gid: this.props.presentation}).observe({
+    Presentations.find({gid: this.props.presentation}).observe({
```

Comment:
I don't think we're using Presentations any more?

Deck stores the slides,
while each time it's run, you create a new show document